src/ESPEasy.ino

```diff
@@ -189,6 +189,13 @@ void setup()
 //  ets_isr_attach(8, sw_watchdog_callback, NULL);  // Set a callback for feeding the watchdog.
 #endif
 
+
+#if FEATURE_ADC_VCC
+  if (!wifiConnectInProgress) {
```

Comment:
@TD-er this condition can be removed here :)

Reply:
My plan is also to include a check for minimum voltage in order to decide whether we need to continue to boot.
Without WiFi enabled, it is drawing less current, so it makes sense the voltage is higher.

Reply:
Removed the check.
Will think about the implications of higher reported voltages.